Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN set -ex\
 		dnsmasq \
 		memcached \
 		nginx \
+		libpq-devel \
```

Comment:
Runtime. This would normally be part of psycopg-binary, but there's no available build for for arm64. This thread has some more info https://github.com/psycopg/psycopg2/issues/1286

Reply:
gotcha - thanks for expanding!